var/spack/repos/builtin/packages/googletest/package.py

```diff
@@ -39,6 +39,8 @@ class Googletest(CMakePackage):
 
     variant('pthreads', default=True,
             description='Build multithreaded version with pthreads')
+    variant('shared', default=False,
+            description='Build shared libraries (DLLs)')
```

Comment:
Most `shared` variants default to True.

Reply:
GTest's build has apparently evolved.  In the beginning, it was a "simple lightweight" thing --- it looked like people built the `gtest.a` library and copied it, plus `gtest.h` header, to whatever place they needed it, inside some other project's build.  It built only a `.a` because they are simpler, and you were "supposed" to copy the library into your own project anyway.

Starting with `@1.8`, it looks like GTest is moving toward a more standard CMake build --- including a way to install itself.  So it's not surprise to me that shared libs are being added.

For this reason, I could go either way on making `+shared` the default.

Reply:
He already gave his opinion.  If you want him to reconsider it, I would recommend asking for that explicitly, in the main comments section for this PR.